docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+
+services:
+  sirbot:
+    image: pyslackers/sirbot:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    restart: unless-stopped
+    expose:
+      - "8080"
```

Comment:
It's not for the host but the nginx. I figured we would only have one nginx for the website and sirbot ?